stream/cards.md

```diff
@@ -143,6 +170,8 @@ https://api.fidel.uk/v1/cards/f76ed1be-e434-480b-aa1d-ff48f548f62a/consents \
     "expMonth": 10,
     "expYear": 2025,
     "countryCode": "USA"
+    "cvc": "123", // required if verificationMethod is 3ds
+    "cardholderName": "John Doe" // required if verificationMethod is 3ds
```

Comment:
Should we rather have two "create consent" sections? One for each method?

Reply:
Also, should we have a "Fingerprint" section as well? That's a new endpoint

Reply:
Do you think it'll make more sense? I'd say that it'll only make the page bigger and this would be clear on the API reference, I think 🤔